cmd/compile/compile.go

```diff
@@ -0,0 +1,69 @@
+package compile
```

Comment:
We have a naming problem. Maybe layering.
Package and filename should be called `compiler`. Also, for some reason I don't understand why this is inside pligos/cmd/compiler. I think you might have just done it too fast. 

There's a pligos/pkg/compiler, is there a reason why this code isn't there? Also, there's Compile here that looks like a helper, and calls the real c.Compile(). :/

Just try to move the contents of this file to pligos/pkg/compiler and that will force you to put things potentially in the right place.